### PR TITLE
fix: Only allow / et. al to be formatted

### PR DESCRIPTION
**Pull request by spydon**

### CODE REVIEW

File: packages/ubuntu_bootstrap/lib/pages/storage/manual/manual_storage_dialogs.dart [hunk 1]
```diff
@@ -187,15 +187,19 @@ Future<void> showEditPartitionDialog(
...
-                  valueListenable: partitionFormat,
-                  builder: (context, value, child) {
+                AnimatedBuilder(
+                  animation:
+                      Listenable.merge([partitionFormat, partitionMount]),
+                  builder: (context, child) {
                     final configFormat = originalPartition != null
                         ? PartitionFormat.fromPartition(originalPartition)
                         : null;
+                    print(partitionMount.value);
```
Comment (d-loose) [inline packages/ubuntu_bootstrap/lib/pages/storage/manual/manual_storage_dialogs.dart:197]: Debugging like a pro! :)

**Review by d-loose — APPROVED** [review]
LGTM :+1:

**Comment by ciampix** [comment]
Please remove this "fix" because it makes my installer life MUCH harder... see this discussion: https://github.com/canonical/ubuntu-desktop-provision/issues/563

**Comment by spydon** [comment]
> Please remove this "fix" because it makes my installer life MUCH harder... see this discussion: #563

There is no point in spreading this discussion over multiple issues and PRs in this repository.

**Pull request merged**